app/assets/stylesheets/application.scss

```diff
@@ -26,6 +26,21 @@
   border-bottom: 1px solid gray;
 }
 
+.card {
+  background-color: #f2580b;
+  border: 0;
+  color: #fff;
+  cursor: pointer;
+  font-size: 18px;
```

Comment:
Properties should be ordered background-color, border, color, cursor, display, font-size, height, transition, width